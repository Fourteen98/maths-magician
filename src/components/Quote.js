function Quote() {
  return (
    <div className="quote">
      <p>

        Mathematics are the result of mysterious powers which no one understands,
        and which the unconscious recognition of beauty must play an important part.
        Out of an infinity of designs a mathematician chooses one pattern for beautyâ€™s
        sake and pulls it down to earth.
        <span className="author"><i>-Marston Morse, American mathematician</i></span>
      </p>
    </div>
  );
}

export default Quote;
