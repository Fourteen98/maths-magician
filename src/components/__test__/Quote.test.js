import React from 'react';
import renderer from 'react-test-renderer';
import Quote from '../Quote';

it('renders to match the snapshot', () => {
  const tree = renderer.create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
