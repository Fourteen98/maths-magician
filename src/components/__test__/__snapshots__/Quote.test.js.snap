// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders to match the snapshot 1`] = `
<div
  className="quote"
>
  <p>
    Mathematics are the result of mysterious powers which no one understands, and which the unconscious recognition of beauty must play an important part. Out of an infinity of designs a mathematician chooses one pattern for beautyâ€™s sake and pulls it down to earth.
    <span
      className="author"
    >
      <i>
        -Marston Morse, American mathematician
      </i>
    </span>
  </p>
</div>
`;
